deploy/3.rateOracles.ts

```diff
@@ -118,6 +118,40 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   }
   // End of Aave Rate Oracles
 
+  // Aave BORROW Rate Oracle
+  if (existingAaveLendingPoolAddress && aaveTokens) {
```

Comment:
Any good reason but to add borrow logic to the identical block above?

Perhaps because we should really be using aaveBorrowConfig? (In Reich case fine to add a new if block but I would also change some variable names)